core/src/main/scala/codesearch/core/index/HackageSources.scala

```diff
@@ -44,7 +44,7 @@ object HackageSources extends Sources[HackageTable] {
       pwd / 'data / 'packages / name / ver / s"$ver.tar.gz"
 
     val packageFileDir =
-      pwd / 'data / 'packages / name / ver / ver
+      pwd / 'data / 'packages / name / ver
```

Comment:
Note: this breaks compatibility with the version running in production. However, we're going to break quite a few other things (configs, paths, etc) so perhaps let's not worry about this.

Once the dust settles, I will deploy Codesearch to a different machine, let it download packages and then change DNS entries to point to the new machine.